packages/utilities/web-vitals/CHANGELOG.md

```diff
@@ -3,3 +3,4 @@
 | Version | Description |
 |---------|-------------|
 | 1.0.0   | [PR#3784](https://github.com/bbc/psammead/pull/3784) Initial creation of package. |
+| 1.0.1   | [PR#3866](https://github.com/bbc/psammead/pull/3866) Rewrite async function to prevent regeneratorRuntime errors |
```

Comment:
```suggestion
| 1.0.1   | [PR#3866](https://github.com/bbc/psammead/pull/3866) Rewrite async function to prevent regeneratorRuntime errors |
| 1.0.0   | [PR#3784](https://github.com/bbc/psammead/pull/3784) Initial creation of package. |
```